system/forms/widgets/iframewidget.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form i18nBaseUri="widgets.iframewidget:">
+	<tab>
+		<fieldset>
+			<field name="url"     required="true" />
+			<field name="border"  control="yesNoSwitch" />
+			<field sortorder="60" name="width"    control="spinner" required="false" default="560" />
```

Comment:
Sorry, missed this before. Could you sort out the sort order on these? (width and height both have sortorder=60 - all fields should have sort order here, 10, 20, 30, 40.